etl/steps/data/garden/aviation_safety_network/2023-04-18/aviation_statistics.py

```diff
@@ -0,0 +1,136 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+from structlog import get_logger
+
+from etl.helpers import PathFinder, create_dataset
+
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+# Select columns.
+COLUMNS = {
+    # Columns that could not be found in spreadsheet from dataframe extracted from their website.
+    "country": "country",
+    "year": "year",
+    "hijackings": "hijacking_incidents",
+    "hijacking_fatalities": "hijacking_fatalities",
+    # Unused columns.
+    # 'airliner_accidents',
+    # 'airliner_fatalities',
+    # 'cargo',
+    # 'charter',
+    # 'corp__jet_accidents',
+    # 'corp__jet_fatalities',
+    # 'domestic_scheduled_passenger',
+    # 'ferry__postioning',
+    # 'intl_scheduled_passenger',
+    # 'training',
+    # Columns from the spreadsheet.
+    "accidents_with_passenger_and_cargo_flights_including_hijacking_etc": "accidents_with_passenger_and_cargo_flights_including_hijacking_etc",
+    "fatalities_with_passenger_and_cargo_flights_including_hijacking_etc": "fatalities_with_passenger_and_cargo_flights_including_hijacking_etc",
+    # Unused columns:
+    # 'accidents_excluding_hijacking_etc',
+    # 'accidents_including_hijacking_etc',
+    # 'accidents_with_corporate_jets',
+    # 'accidents_with_passenger_flights_including_hijacking_etc',
+    # 'fatalities_excluding_hijacking_etc',
+    # 'fatalities_including_hijacking_etc',
+    # 'fatalities_with_corporate_jets',
+    # 'fatalities_with_passenger_flights_including_hijacking_etc',
+}
+
+# Select columns from WDI table, and how to rename them.
+WDI_COLUMNS = {
+    "country": "country",
+    "year": "year",
+    "is_air_dprt": "departures_worldwide",
+    "is_air_psgr": "passengers_carried",
+}
+
+
+def run(dest_dir: str) -> None:
```

Comment:
In general, I lean towards defining functions for the different code sections. E.g. `process_wdi_data`.

Just a preference. Most important are comments, and that's here!